File: app/src/main/java/com/FaceLab/FaceLab/InfoActivity.java
package com.FaceLab.FaceLab;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;

public class InfoActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_info);
    }

    public void customOnClick(View v) {
        if (v.getId() == R.id.info_btn_back)
            finish();
    }
}

File: app/src/main/java/com/FaceLab/FaceLab/InfoActivity.kt
package com.FaceLab.FaceLab

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class InfoActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_info)
    }

    fun customOnClick(v: View) {
        if (v.id == R.id.info_btn_back) finish()
    }
}